### ISSUE TITLE
bug: sending tokens from eth to cosmos

### ISSUE THREAD

**Issue by alexanderbez** [PeggyJV/gravity-bridge#130]
1. Deploy an ERC20 contract of a token that has metadata where display has exponent 6. e.g. (usomm:0, somm:6)
2. Send 300usomm tokens from cosmos to eth

```shell
$ <appd> tx gravity send-to-etheruem <ethAddr> 300usomm 3usomm ...
```

3. Query eth balance and ensure it's correct -- should be 300usomm
4. Try to send back the same 300 from Ethereum to Cosmos:

```
Insufficient balance 300000000 > 300', client/src/main.rs:301:13
```

It seems like decimal conversions and/or assumptions continue to bite us. I'm wondering if we should remove decimal usage all together and always use/assume 0. This is what Injective does I believe.

/cc @levicook

**Comment by alexanderbez** [comment]
Narrowed it down the fact that we call `fraction_to_exponent`. We need to remove this utility and all references to it and just send raw user inputed amounts.

**Comment by alexanderbez** [comment]
closed by #132
